src/webui/webui.pri

```diff
@@ -19,7 +19,7 @@ SOURCES += \
     $$PWD/qtorrentfilter.cpp \
     $$PWD/abstractwebapplication.cpp
 
-# QJson JSON parser/serializer for using with Qt4
-lessThan(QT_MAJOR_VERSION, 5): include(qjson/qjson.pri)
+# QJson JSON parser/serializer for using with Qt < 5.5
+lessThan(QT_MAJOR_VERSION, 5) | lessThan(QT_MINOR_VERSION, 5): include(qjson/qjson.pri)
```

Comment:
With an AND it fails in some cases, with an OR it fails wit Qt > 6. I modified the code to work in any case, even if Qt6 does not exist yet.